packages/admin/admin/src/hooks/useCurrentDeviceType.ts

```diff
@@ -0,0 +1,10 @@
+export enum DEVICE_TYPE {
+    MOBILE = "mobile",
+    DESKTOP = "desktop",
+}
+
+export function useCurrentDeviceType() {
```

Comment:
There's no need for this to be a hook, as the user agent will not change. We would only need a hook if we support SSR, but as we don't plan on doing that for Admin application in the near future, this should be a simple constant.

And I'd use a package for that, which has an up-to-date user agent: https://www.npmjs.com/package/is-mobile

Reply:
Done